sno/geometry.py

```diff
@@ -396,19 +400,19 @@ def geom_to_ewkb(gpkg_geom):
     ewkb_geom_type = wkb_geom_type
     ewkb_geom_type |= 0x80000000 * has_z
     ewkb_geom_type |= 0x40000000 * has_m
-    ewkb_geom_type |= 0x20000000 * (srid > 0)
+    ewkb_geom_type |= 0x20000000 * (crs_id > 0)
 
     ewkb = struct.pack(f"{bo}BI", int(wkb_is_le), ewkb_geom_type)
 
-    if srid > 0:
-        ewkb += struct.pack(f"{bo}I", srid)
+    if crs_id > 0:
+        ewkb += struct.pack(f"{bo}I", crs_id)
 
     ewkb += gpkg_geom[(wkb_offset + 5) :]
 
     return ewkb
 
 
-def hexewkb_to_geom(hexewkb):
+def hexewkb_to_gpkg_geom(hexewkb):
     """
```

Comment:
this method doesn't match up with what we decided on wrt geometry formatting. eg. envelopes & endianness

Reply:
Fixed by calling normalise_gpkg_geometry. There might be a more efficient way to do it: added a TODO